src/main/java/underdogs/devbie/auth/controller/interceptor/BearerAuthInterceptor.java

```diff
@@ -19,6 +20,11 @@
 
     @Override
     public boolean preHandle(HttpServletRequest request, HttpServletResponse response, Object handler) {
+        InterceptorValidator interceptorValidator = new InterceptorValidator();
+
+        if (!interceptorValidator.isValid(handler)) {
+            return true;
+        }
```

Comment:
별건 아니지만, `!` 논리식을 지양하고 싶어서 의견 남겨봅니다.
```suggestion
        if (interceptorValidator.isNotValid(handler)) {
            return true;
        }
```

Reply:
감사합니다 !